eap-aro/src/main/scripts/jboss-setup.sh

```diff
@@ -382,7 +382,7 @@ consoleUrl=$(az aro show -g $RESOURCE_GROUP -n $CLUSTER_NAME --query 'consolePro
 
 # Install the OpenShift CLI
 downloadUrl=$(echo $consoleUrl | sed -e "s/https:\/\/console/https:\/\/downloads/g")
-wget ${downloadUrl}amd64/linux/oc.tar -q -P ~
+wget --no-check-certificate ${downloadUrl}amd64/linux/oc.tar -q -P ~
```

Comment:
It would have been better to separate this bug fix from a PR that is otherwise only about testing.